src/apiCalls.tsx

```diff
@@ -48,7 +48,8 @@ const cleanArticles = (articles: OriginalArticle[]): CleanedArticle[] => {
       abstract,
       short_url,
       sentiment: 0,
-      multimedia: multimedia[0]
+      multimedia: multimedia[0],
+      id: Math.floor(Math.random() * 100),
```

Comment:
I needed an id for Router, so I added it here for each article. Date.now() was giving them all the same id as it was being generated in the same second.

Reply:
what if we are getting more than 999 articles back? Just kidding (and this is no place for jokes).

Reply:
How about using the article title for the id? Might need to do some data cleaning, so maybe not worth it.

Reply:
I was using that originally, but then the url gets really long when we use Router. Do you all mind a really long url?

Reply:
You could create a substring of only the first handful of words.